tests/tasks/presetup.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Is this something we are going to reuse in multiple playbooks?
If we are only going to use on `tests/tests_fs_attrs.yml` I would just define it there.